tailwind.config.js

```diff
@@ -18,14 +18,16 @@ module.exports = {
       h5: ['20px', '22px'],
       h4: ['22px', '24px'],
       h3: ['24px', '26px'],
-      h2: ['32px', '36px'],
-      h1: ['34px', '48px'], // requires red <hr /> below
+      h2: ['36px', '40px'],
+      xs: ['32px', '36px'],
+      h1: ['38px', '42px'], // requires red <hr /> below
```

Comment:
These changes are global, and affect the whole application - I am not sure if this is intended (or good). If it is intended, then I suppose that's fine. If not, could we see if we can apply these styles within the component, rather than globally?

Reply:
I think we need those changes. The change for h1/h2 font is to make sure styles align with final mockups, where h1 is defined as font size 38px, line-height 42, and h2 has 36px font size and 40px line height